.github/workflows/testChecklists.yml

```diff
@@ -0,0 +1,16 @@
+name: "Contributor Checklist"
+
+on:
+  issue_comment:
+    types: [created, edited]
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  checklist:
+    runs-on: ubuntu-latest
+    steps:
+      - name: contributorChecklist.js
+        uses: Expensify/App/.github/actions/javascript/contributorChecklist@andrew-checklist
```

Comment:
Just for my own understanding, would you update this to `@main` in a followup PR?

Reply:
Yes, I could even update it in this PR once we verify it works.